.github/actions/buildout/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'buildout'
+description: 'Install python, virtualenv and buildout'
+inputs:
+  python-version:
+    required: true
+  plone-version:
+    required: true
+  github-token:
+    required: true
```

Comment:
I would say `github-token` should be optional.  That is how it is in [`actions/setup-python`](https://github.com/actions/setup-python/blob/v4.6.0/action.yml#L18-L20)